OpenRA.Mods.Cnc/UtilityCommands/ImportGen2MapCommand.cs

```diff
@@ -356,21 +126,20 @@ static void ReadTiles(Map map, IniFile file, int2 fullSize)
 
 				var dx = rx - ry + fullSize.X - 1;
 				var dy = rx + ry - fullSize.X - 1;
```

Comment:
this pattern is commonly repeated, perhaps add another ToMPos function with these inputs?